src/site/stages/build/options.js

```diff
@@ -3,12 +3,12 @@
 const path = require('path');
```

Comment:
We're kind of burying this file in the src folder. I get the way you've laid things out, but I wonder if the extra stages directory buys us a lot. There's a lot of nesting, might be good to see if we can reduce that a little and still be better organized.

I say this partly because I find the deeply nested code in `src/platform` pretty annoying at times (even though it was me that put it that way).

Reply:
I really like that idea, but I'm really struggling to think of a good location to contain what is currently in `stages` 🤔